Lib9c/Action/ActionRenderer.cs

```diff
@@ -10,25 +13,124 @@
 
 namespace Nekoyume.Action
 {
-    public class ActionRenderer
+    using Action = PolymorphicAction<ActionBase>;
```

Comment:
I believe it may cause confused it with `System.Action`. How about using another alias name like `ActionType` or a better one not confused?

Reply:
@moreal Renamed to `NCAction`: 43f06784c15bad1538451a5e96288bdb01d4c571.